IanGRap/Squish
# Player animations 2

Contributors: Ian, Alfredo 

Goal: Add Animations to the player character. 

Important Files: 
GooController.controller: Animation controller for the goo part of the player
SkeleController.controller: Animation controller for the skele part of the player
PlayerAnimationDelegator.cs: Allows for changes to the Animator controller parameters\
PlayerDash.cs / PlayerJump.cs: Now call the PlayerAnimationDelegator
TemporaryRotator.cs: changed values to work with new player model\

How to Test: 
In the 2017 scene. Run the game and move the player around. The player should animate along with the movement. 

* Notes: These animations are a first pass and will be revised in the future. 
* We had to destroy and recreate the player prefab because we wanted to remove the previous model
